tests/conftest.py

```diff
@@ -106,3 +106,24 @@ def mock_openzwave():
         'openzwave.group': base_mock.group,
     }):
         yield base_mock
+
+
+@pytest.fixture
+def mock_device_tracker_conf():
+    """Prevent device tracker from reading/writing data."""
+    devices = []
+
+    async def mock_update_config(path, id, entity):
+        devices.append(entity)
+
+    with patch(
+        'homeassistant.components.device_tracker'
+        '.DeviceTracker.async_update_config',
+            side_effect=mock_update_config
+    ), patch(
+        'homeassistant.components.device_tracker.async_load_config',
+            side_effect=lambda *args: mock_coro(devices)
```

Comment:
Are lambda function coroutines/do the inherit it in some way?

Lambda functions does not seem to be async yet according to PEP492

Reply:
Async functions return coroutines when called. These can be awaited to execute it and get the result. So in this case the lamda passes the coroutine to the caller.